.github/workflows/porter_app_worker_4828.yml

```diff
@@ -18,18 +18,9 @@ jobs:
             - name: Setup porter
               uses: porter-dev/setup-porter@v0.1.0
 
-            - name: Install Poetry
-              uses: snok/install-poetry@v1
-              with:
-                version: 1.8.5
-
             - name: Build Dist
               run: make full-dist
 
-            - name: Export Requirements
-              run: |
-                cd arcade && poetry export --output ../dist/requirements.txt
-
```

Comment:
I don't think porter needs requirements.txt?

Reply:
well if we just install the whl file, do we still need reqs?

Reply:
updated Dockerfile - requirements.txt no longer needed